src/components/Navbar.jsx

```diff
@@ -14,6 +16,7 @@ function Navbar() {
       </Link>
       <ul className="nav-items">
         {links.map((link) => <li key={link.text} className="nav-item"><NavLink to={link.path}>{link.text}</NavLink></li>)}
+        <Button title={<CgProfile />} handleClick={null} className="btn-icon" />
```

Comment:
- [ ] Please make the background color of your navbar to be `#ffffff` just like the Zeplin design template.